content/00.front-matter.md

```diff
@@ -2,14 +2,22 @@
 
 {% for author in authors %}
 + **{{author.full_name}}**<br>
+  ![ORCID icon](images/orcid.svg){height="13px"}
```

Comment:
We could also check whether orcid is defined and not print the symbol? This would leave only a preceeding interpunct which isn't too big of a deal. Or we could print `missing ORCID` next to the symbol if we really want to nudge authors to use ORCID.